shared/src/commonMain/kotlin/io/projectnewm/login/LoginFieldValidator.kt

```diff
@@ -0,0 +1,28 @@
+package io.projectnewm.login
+
+class LoginFieldValidator {
+    public companion object {
+        public fun validate(email: String, password: String): Boolean {
```

Comment:
No need for the companion object - 

```suggestion
object LoginFieldValidator {
    public fun validate(email: String, password: String): Boolean {
        .....
```